wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -150,6 +155,39 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
 			<!--===============================-->
+			<xs:complexType name="AudioClipCapabilities">
+				<xs:sequence>
+					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first ONVIF then Vendor -->
+				</xs:sequence>
+				<xs:attribute name="MaxAudioClipLimit" type="xs:int">
+					<xs:annotation>
+						<xs:documentation> Indicates the maximum number of audio clips that can be uploaded to the device.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="MaxAudioClipSize" type="xs:float">
+					<xs:annotation>
+						<xs:documentation> Indicates the maximum size of the audio clip that can be uploaded, in KB.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="SupportedAudioClipFormat" type="tt:StringList">
```

Comment:
Is it possible that device supports different formats for different Audio Outputs? If yes, then this capabilities approach will not work. A Client does not have possibility to detect which type with which audio output works.

Reply:
all audio outputs of the device will support same audio clip formats.